funding/commitment_type_negotiation.go

```diff
@@ -360,6 +360,7 @@ func hasFeatures(local, remote *lnwire.FeatureVector,
 
 	for _, feature := range features {
```

Comment:
note that the commit message does not match the commit contents

Reply:
Can I ask what kind of commit are you referring? :)

Reply:
Ok I now understand, and I think with https://github.com/lightningnetwork/lnd/commit/3c4caadf18cdd2e54a2155b1095c3da76507a022 the commit body is in line with the code, is it? or do we need a rewording there to?